app/screens/home/search/initial/recent_searches/recent_item.tsx

```diff
@@ -33,16 +56,30 @@ const RecentItem = ({item, setRecentValue}: Props) => {
     }, [item.id, serverUrl]);
 
     return (
-        <OptionItem
-            action={handlePress}
-            icon={'clock-outline'}
-            inline={true}
-            label={item.term}
-            onRemove={handleRemove}
-            testID={'search.recent_item'}
-            type='remove'
```

Comment:
If I remember correctly, The remove type was a new one added by Jason. Has to be removed? I think this is the only place where it was being used.

Reply:
indeed - corrected